doc/code/qml_qchem.rst

```diff
@@ -63,21 +63,7 @@ We then construct the Hamiltonian.
     hamiltonian, qubits = qml.qchem.molecular_hamiltonian(molecule, args=args)
 
 >>> print(hamiltonian)
-  (-0.35968235922631075) [I0]
-+ (-0.11496335836166222) [Z2]
-+ (-0.11496335836166222) [Z3]
-+ (0.13082414303722753) [Z1]
-+ (0.13082414303722759) [Z0]
-+ (0.1031689825163302) [Z0 Z2]
-+ (0.1031689825163302) [Z1 Z3]
-+ (0.15329959994281844) [Z0 Z3]
-+ (0.15329959994281844) [Z1 Z2]
-+ (0.15405495855815063) [Z0 Z1]
-+ (0.1609686663987323) [Z2 Z3]
-+ (-0.05013061742648825) [Y0 Y1 X2 X3]
-+ (-0.05013061742648825) [X0 X1 Y2 Y3]
-+ (0.05013061742648825) [Y0 X1 X2 Y3]
-+ (0.05013061742648825) [X0 Y1 Y2 X3]
+-0.3596823592263396 * I(0) + 0.1308241430372839 * Z(0) + -0.11496335836158356 * Z(2)+ 0.10316898251630022 * (Z(0) @ Z(2)) + 0.1308241430372839 * Z(1) + 0.15405495855812648 * (Z(0) @ Z(1)) + 0.050130617426473734 * (Y(0) @ X(1) @ X(2) @ Y(3)) + -0.050130617426473734 * (Y(0) @ Y(1) @ X(2) @ X(3)) + -0.050130617426473734 * (X(0) @ X(1) @ Y(2) @ Y(3)) + 0.050130617426473734 * (X(0) @ Y(1) @ Y(2) @ X(3)) + -0.11496335836158356 * Z(3) + 0.15329959994277395 * (Z(0) @ Z(3)) + 0.10316898251630022 * (Z(1) @ Z(3)) + 0.15329959994277395 * (Z(1) @ Z(2)) + 0.16096866639866414 * (Z(2) @ Z(3))
```

Comment:
This is correct for the code as it is now, though I prefer the multi-line style. I wonder if this is a regression. @Qottmann do you remember, when we worked on the improved op repr, if the improvements (including multi-line style) extended to print statements?

Reply:
This is coming from using the `__str__` (from explicit `print`) versus using `__repr__` (multiline) for `CompositeOps`. I would suggest to just have `>>> hamiltonian` instead of `print(hamiltonian)` which then gives the nicer output.